src/test/java/net/syscon/elite/api/resource/impl/OffenderMovementsResourceImplIntTest_getCourtHearings.java

```diff
@@ -0,0 +1,116 @@
+package net.syscon.elite.api.resource.impl;
+
+import net.syscon.elite.api.model.ErrorResponse;
+import net.syscon.elite.executablespecification.steps.AuthTokenHelper;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Test;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpMethod;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class OffenderMovementsResourceImplIntTest_getCourtHearings extends ResourceTest {
+
+    @Test
+    public void get_court_hearings_for_booking_returns_no_court_hearings() {
```

Comment:
Ruby? :trollface:

Reply:
Someone noticed...I have to be honest I prefer this to Java convention. https://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html